api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ESMonitoringManager.java

```diff
@@ -93,12 +97,14 @@ public InputStream getStatsForNodeAsInputStream(final String nodeName,
                                           ? minDuration
                                           : interval;
         final List<MonitoringStats> monitoringStats = getStats(nodeName, start, end, adjustedDuration);
-        final MonitoringStatsWriter statsWriter = new MonitoringStatsWriter();
-        try {
-            return new StringInputStream(statsWriter.convertStatsToCsvString(monitoringStats));
-        } catch (IOException e) {
-            throw new IllegalStateException(messageHelper.getMessage(MessageConstants.ERROR_BAD_STATS_FILE_ENCODING),
-                                            e);
+        switch (type) {
```

Comment:
Could we use approach similar to the one used in `CloudFacadeImpl` to determine what kind of writer is required?

Reply:
Done